os/arch/arm/src/amebasmart/amebasmart_smp.c

```diff
@@ -201,15 +194,21 @@ void smp_init(void)
 #if ( CONFIG_SMP_NCPUS > 1 )
 	for (xCoreID = 0; xCoreID < CONFIG_SMP_NCPUS; xCoreID++) {
 		if (xCoreID == up_cpu_index()) {
-			// pmu_set_secondary_cpu_state(xCoreID, CPU1_RUNNING);
+			up_set_secondary_cpu_state(xCoreID, CPU_RUNNING);
 			continue;
 		}
-
 		err = psci_cpu_on(xCoreID, (unsigned long)__cpu1_start);
 		if (err < 0) {
-			printf("CPU%d: failed to boot: %d\n", (int)xCoreID, (int)err);
+			dbg("CPU%d: failed to boot: %d\n", (int)xCoreID, (int)err);
```

Comment:
Is it ok to just print the error msg here and proceed?
When we return from this API, we assume that the second core has booted up.

Reply:
Your point make sense, the operation is irreversible (ie. unrecoverable). So, we should make a crash here.